modules/Components/src/client/ACASFormChemicalStructure.coffee

```diff
@@ -95,6 +95,12 @@ class MaestroChemicalStructureController extends Backbone.View
 	setMol: (molStr) ->
 		@maestro.sketcherImportText molStr
 
+	isEmptyMol: (molStr) ->
```

Comment:
hypernit / comment: Could also combine this with "getMol" so it doesn't need any input arguments. But I think this is A-OK the way it is.